src/components/Osm/Editor.jsx

```diff
@@ -131,53 +123,53 @@ class Editor extends React.Component {
     return '';
   }
 
-  addField(tag, value) {
+  addField = (tag, value) => {
```

Comment:
Oh, i wasn't aware we were using class properties...

Reply:
No worries, it was just to be less verbose